packages/api/src/command/medical/cq-directory/parse-cq-directory-entry.ts

```diff
@@ -77,16 +79,17 @@ function getUrls(contained: Contained): XCUrls | undefined {
   return urls;
 }
 
-function getCoordinates(address: Address[] | undefined): { lat: string; lon: string } | undefined {
+function getCoordinates(address: Address[] | undefined): { lat: number; lon: number } | undefined {
   if (!address) return;
   const coordinates = address.flatMap(a => {
```

Comment:
Since we are returning the first match anyway, use `find` instead of `flatMap` to directly return the first matching element from the address array and save the unnecessary subsequent iterations